pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/scaleup-scaledown/index.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: layout.pug
+navigationTitle: Scale Up or Down a Cluster Node
+title: Scale Up or Down a Cluster Node
+menuWeight: 84
+excerpt: Procedure to scale-up or scale-down nodes in an existing DKP cluster.
+beta: false
+enterprise: false
+---
+
+## Prerequisites
+
+- You must have the bootstrap node running with the SSH key/secrets created.
+- The export values in the environment variables section should contain the addresses of the nodes that you need to add in the section [define infrastructure][define-infrastructure].
+- Update the preprovisioned_inventory.yaml with the new host addresses.
+- Run the kubectl apply command.
+
+## Scale up a Cluster Node
+
+Follow these steps:
+
+1.  Fetch the existing preprovisioned_inventory:
+
+    ```bash
+    $ kubectl get preprovisionedinventory
+    ```
+
+1.  Edit the preprovisioned_inventory to add additional IPs needed for additional worker nodes in the spec.hosts: section: 
+
+    ```bash
+    $ kubectl edit preprovisionedinventory <preprovisioned_inventory> -n default 
+    ```
+
+1.  (Optional) Add any additional IPs that you require:
+
+    ```bash
+    spec: 
+        hosts: 
+        - address: <worker.ip.add.1> 
+        - address: <worker.ip.add.2> 
+    ```
+
+    After you edit preprovisioned_inventory, fetch the machine deployment. The naming convention with md means that it is for worker machines.\
+    For example:
+
+    ```bash
+    $ kubectl --kubeconfig ${CLUSTER_NAME}.conf get machinedeployment 
```

Comment:
For this section we should separate out commands the user is expected to run from the output, also removing $ at the beginning of kubectl commands. This way the user clearly understands what they should copy and run against their clusters.